### ISSUE TITLE
Project Dead?

### ISSUE THREAD

**Issue by wzrdtales** [saneki/Xotic#2]
I guess this project is dead? It seems not to work anymore, is there any intention to continue the development?

**Comment by saneki** [comment]
Yep, pretty much dead. I might update it in the future, maybe when toxcore supports new groupchats, but there hasn't really been any demand.

**Comment by rodneyrod** [comment]
I'd be interseted in beta testing any new changes, I've been looking for a Tox solution that I can use on my remote server, that doesn't use any Go code since that is a royal bitch to setup the workspace and get running.

**Comment by saneki** [comment]
@rodneyrod If you like/don't mind consoles, you could try ssh + [toxic](https://github.com/JFreegman/toxic). No Go code, just plain ol' C.

If you're not a console fan, well.. is there a reason you need a P2P messenger running on a server anyways?

**Comment by rodneyrod** [comment]
@saneki I have a remote server and my terrible internet is too unstable to keep a Tox connection open for longer than 20-30 minutes. SSH isn't an option on some of the computers I have to use.
I already use Node for another webapp so I thought this would be the best option.

So yeah, there IS a reason why I want this to run on a server.
